linera-sdk/src/lib.rs

```diff
@@ -382,6 +382,24 @@ pub struct OutgoingMessage<Message> {
     pub message: Message,
 }
 
+impl<Message> OutgoingMessage<Message>
+where
+    Message: Serialize,
+{
+    /// Serializes the internal `Message` type into raw bytes.
+    pub fn into_raw(self) -> OutgoingMessage<Vec<u8>> {
+        let message = bcs::to_bytes(&self.message).expect("Failed to serialize message");
```

Comment:
We have a long standing issue about this (see #147 and #743): BCS serialization can actually fail (for some Rust types). It would be nice to address it.

Reply:
Right, but panicking is the answer, right? Although I still need to remove the `Result` return types from the traits.